ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -280,6 +280,24 @@
    */
   CompletableFuture<ConnectorDescription> describeConnector(String connectorName);
 
+  /**
+   * Define a session variable which can be referenced in sql commands by wrapping it with ${}
+   *
+   * @param variable the name of the variable
+   * @param value the value represented by the variable
+   */
+  void define(String variable, Object value);
+
+  /**
+   * Undefine a variable.
```

Comment:
See above. (Meta-note: The purpose of these javadocs is so that a user unfamiliar with the client can understand what the methods do and how to use them at a glance. Better to be more verbose than less :) )